src/MigrationsServiceProvider.php

```diff
@@ -28,11 +28,9 @@ class MigrationsServiceProvider extends ServiceProvider
      */
     public function boot()
     {
-        if (!$this->isLumen()) {
-            $this->publishes([
-                $this->getConfigPath() => config_path('migrations.php'),
-            ], 'config');
-        }
+        $this->publishes([
+            $this->getConfigPath() => config_path('migrations.php'),
```

Comment:
This won't work on a clean Lumen install, as it doesn't know the `config_path` helper.